docs/process.rst

```diff
@@ -7,7 +7,7 @@ Processes
 In Nextflow, a **process** is the basic processing primitive to execute a user script.
 
 The process definition starts with the keyword ``process``, followed by process name and finally the process body
-delimited by curly brackets. The process body must contain a string which represents the command or, more generally,
+delimited by curly braces. The process body must contain a string which represents the command or, more generally,
```

Comment:
Is this a US / British English thing? When I see the word braces I think of the things that hold your trousers up 😅 

(no problem with the change, just curious)

![braces](https://media.giphy.com/media/11BefSUFVYvwiY/giphy.gif)

Reply:
Lol, I don't know, I mostly associate braces with `{}` and brackets with `[]` (or angle brackets for `<>`)

In the US we call them suspenders :smile: